src/extend-node-type.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const { GraphQLInt, GraphQLJSON, GraphQLString } = require('gatsby/graphql');
+
+const { resolveGatsbyImageData } = require('./gatsby-plugin-image');
+
+exports.setFieldsOnGraphQLNodeType = async ({ type, cache, reporter }, configOptions) => {
+  const typePrefix = configOptions.type_prefix || 'Contentstack';
+  if (type.name !== `${typePrefix}_assets`) {
+    return {};
+  }
+
+  // gatsby-plugin-image
+  const getGatsbyImageData = async () => {
+    const { getGatsbyImageFieldConfig } = await import('gatsby-plugin-image/graphql-utils');
+
+    const fieldConfig = getGatsbyImageFieldConfig(
+      async (...args) => resolveGatsbyImageData(...args, { cache, reporter }),
```

Comment:
This is a bit of a nit: Spreading the args here seems like it could be flaky in the case that additional args are ever passed to it. What about something like this instead since only two of those arguments seem to used in `resolveGatsbyImageData` anyways? (Obviously the function signature for `resolveGatsbyImageData` would have to change as well.)

```js
async (image, options) => resolveGatsbyImageData({ image, options, cache, reporter }),
```